src/main/java/recipeio/commands/FindCommand.java

```diff
@@ -0,0 +1,84 @@
+package recipeio.commands;
+
+import recipeio.recipe.Recipe;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class FindCommand {
+    public static void findKeyword(String keyword, ArrayList<Recipe> recipes) {
+        ArrayList<Recipe> matches = new ArrayList<>();
+        if (recipes.isEmpty()) {
+            System.out.println("Sorry, you have no recipes to find matches with. Try adding some!");
+        } else {
+            for (Recipe recipe : recipes) {
+                if (recipe.name.contains(keyword)) {
+                    matches.add(recipe);
+                }
+            }
```

Comment:
Do consider using .stream() methods to reduce the length of method.

You can refer to this repo on how lambda expressions are used.
https://github.com/yuki-zmstr/tasks/blob/master/src/main/java/se/edu/streamdemo/Main.java#L62